model.py

```diff
@@ -1519,9 +1522,8 @@ def bulk_add(self, works, operation, timestamp=None,
         # The SELECT part of the INSERT...SELECT query.
         new_records = _db.query(
             Work.id.label('work_id'), 
-            literal(operation, type_=BaseCoverageRecord.status_enum).label('operation'),
+            literal(operation, type_=String(255)).label('operation'),
```

Comment:
What's the motivation behind this change?

Reply:
I made this change when I was exploring bulk inserts. Type is supposed to be the target column type, and the column type of CoverageRecord.operation isn't an status enum, it's String.

If it's working just fine as-is, I can remove it.

Reply:
You're right, I mixed up operation and status originally. Thanks for fixing my error!